assets/background.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can you please rename the file to `launcher-background-34c3.png` so it is easier to associate it with its product flavor?

Reply:
sure thing!